package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@asyncapi/avro-schema-parser",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
please remove this change, our automated pipeline is handling bumps in `package.json`